src/com/facebook/buck/apple/project_generator/SchemeGenerator.java

```diff
@@ -107,6 +115,7 @@ public SchemeGenerator(
     this.remoteRunnablePath = remoteRunnablePath;
     this.actionConfigNames = actionConfigNames;
     this.targetToProjectPathMap = targetToProjectPathMap;
+    this.additionalSchemeActions = additionalSchemeActions;
```

Comment:
You're not using this value to write data to the file yet?

Reply:
No not yet, wanted to get feedback first on if the type for `additionalSchemeActions` would be acceptable.